### PR TITLE
IME-6 , Update sdk scheme adapter to v23.6.0-snapshot.2

### PR DESCRIPTION
**Pull request by Ujjwal-Izyane**
1. Update the image of the sdk-scheme-adapter in docker-compose.yaml
2. Update ENV variables for the sdk-scheme-adapter in docker compose
3. Add fspiop 2.0 api to spec files
4. Update callback rules for payer
5. Add api definition for sdk-scheme-adapter api v 2.1 backend and outbound
6. Add currency conversion test case for payer side
7. Add document with steps to deploy using docker.
8. Update Rules to simulate FXP side flow

### CODE REVIEW

File: docker-compose/docker-compose.yaml [hunk 1]
```diff
@@ -209,6 +212,10 @@ services:
       - PM4ML_ENABLED=${PM4ML_ENABLED:-false}
       - ENABLE_BACKEND_EVENT_HANDLER=false
       - ENABLE_FSPIOP_EVENT_HANDLER=false
+      - FSPIOP_API_SERVER_MAX_REQUEST_BYTES=209715200
+      - BACKEND_API_SERVER_MAX_REQUEST_BYTES=209715200
+      - SUPPORTED_CURRENCIES=''
```
Comment (vijayg10) [inline docker-compose/docker-compose.yaml:217]: You may need this field to be set when you are working on payee side implementation.

File: testing-toolkit/collections/payer-tests/outbound_transfer.json [hunk 2]
```diff
@@ -1,195 +0,0 @@
-{
```
Comment (vijayg10) [inline testing-toolkit/collections/payer-tests/outbound_transfer.json:1]: Why this file is deleted?

Comment (Ujjwal-Izyane) [reply testing-toolkit/collections/payer-tests/outbound_transfer.json:1]: This file is not deleted , just renamed to the outbound_transfer_fx.json , with an added test case to it

Comment (vijayg10) [reply testing-toolkit/collections/payer-tests/outbound_transfer.json:1]: I think we should have this file intact and have another file for fx usecase.

Comment (vijayg10) [reply testing-toolkit/collections/payer-tests/outbound_transfer.json:1]: Normal transfer and fx transfers are different and we should have both files.

Comment (Ujjwal-Izyane) [reply testing-toolkit/collections/payer-tests/outbound_transfer.json:1]: Okay let me separate the transfers in different files then

Comment (Ujjwal-Izyane) [reply testing-toolkit/collections/payer-tests/outbound_transfer.json:1]: Done

**Review by vijayg10 — APPROVED** [review]

**Pull request merged**